report/src/main/webapp/app/layout/main/test/mainView.test.view.html

```diff
@@ -17,7 +17,7 @@
     limitations under the License.
 
 -->
-<div class="main page-main">
+<div class="page-main">
```

Comment:
'main' is also used with 'toolbar-holder', so expanding and collapsing works properly